src/checkout-buttons/strategies/paypal-button-strategy.spec.ts

```diff
@@ -84,7 +88,8 @@ describe('PaypalButtonStrategy', () => {
         strategy = new PaypalButtonStrategy(
             store,
             paypalScriptLoader,
-            formPoster
+            formPoster,
+            ''
```

Comment:
If we are allowing the value to be `undefined`, all these tests should pass without an empty string.

Reply:
@davidchin so this is an area I'm not entirely sure about. Before my change, the behavior when you didn't pass a `host` was `undefined` since `options && options.host` returns `undefined` when `host` isn't provided. This is now more explicit by the fact that we're capturing it into a variable. WDYT?

Reply:
I think what Ignacio meant was because the last parameter - `host` - is optional, you don't need to make changes to the tests and they will still pass.

Reply:
Right, I was meaning overall I guess, rather than specific to this test.